crypto/random/rand_test.go

```diff
@@ -82,26 +82,33 @@ func TestChacha20Compliance(t *testing.T) {
 	})
 }
 
+func seedMathRand(t *testing.T) {
+	r := time.Now().UnixNano()
+	rand.Seed(r)
+	t.Logf("math rand seed is %d", r)
+}
+
 // The tests are targeting the PRG implementations in the package.
 // For now, the tests are only used for Chacha20 PRG, but can be ported
 // to test another PRG implementation.
 
 // Simple unit testing of Uint using a very basic randomness test.
 // It doesn't evaluate randomness of the output and doesn't perform advanced statistical tests.
 func TestUint(t *testing.T) {
+	seedMathRand(t)
 
 	seed := make([]byte, Chacha20SeedLen)
-	crand.Read(seed)
+	rand.Read(seed)
```

Comment:
why replacing crypto/rand with math/rand?

Reply:
`math/rand` produces deterministic randoms depending on the initializing seed, while `crypto/rand` doesn't allow that. This is useful when a test fails, and by logging the seed we can reproduce the fail. 

Note that `math/rand` shouldn't be used outside of tests when secure entropy is needed.